### ISSUE TITLE
The node to be removed is no longer a child of this node.

### ISSUE THREAD

**Issue by DavidGumm** [olifolkerd/tabulator#2247]
**Describe the bug**
When double clicking on an empty cell I get the error: "tabulator.min.js:4 Uncaught (in promise) DOMException: Failed to set the 'innerHTML' property on 'Element': The node to be removed is no longer a child of this node. Perhaps it was moved in a 'blur' event handler?", It does not happen when the cell has a value and I am removing it, or altering it. If the cell started as null, then when you double click on it will sometimes not take the double click and not add a value causing you to have to double click on the cell twice to get it to add a value.

**Tabulator Info**
- 4.3

**Working Example**
https://codepen.io/davidgumm/pen/jgyjZN?editors=1010

**To Reproduce**
1. Go to any cell in the columns  Start, and Stop.
2. Double click on 'the cell this will remove the value, then double click on it again.
3. See error

**Expected behavior**
I except the error not to get thrown.

**Desktop (please complete the following information):**
 - OS: Windows 10/ Mac OS
 - Browser: chrome
 - Version 75

**Additional context**
If you need any additional information please let me know. Thanks for build a great product.

**Comment by olifolkerd** [comment]
Thanks for the headsup,

I will look into this for the next release.

Cheers

Oli :)

**Comment by DavidGumm** [comment]
Thanks a ton! I was worried I was doing something wrong.

**Comment by olifolkerd** [comment]
Hey @DavidGumm 

Looking into this further i dont think it is an issue with Tabulator.

If you look at the stack trace of the console log the error is happening in the EditedTimeSpan. a bit of console logging later and i would say that the "parsed" variable you are passing into the moment duration is undefined, which is an invalid value to pass into that function which is causing the error.

I hope that helps,

Cheers

Oli :)

**Comment by DavidGumm** [comment]
That does help a lot. I will check on it Monday. Thanks.

**Comment by DavidGumm** [comment]
I was able to get to a computer and check on the code. I added a check for undefined. It works consistently with that check there but still throws an error every time the value is blank and I double click on it.

**Comment by olifolkerd** [comment]
Can you check the error on the stack trace. Where is it comming from

**Comment by prsanjay** [comment]
Hi @olifolkerd , I am also facing the same issue. When user clicks on cell in the call back I make the cell value empty by calling `cell.setValue("", true);`

When I call this, in tabulator.js - version 4.7 throws same above exception. Line no-7211 in tabulator.js

Basically, I set value to input initially which we consider as place holder like (Enter Name) and when user clicks on cell, I set the blank value using `cell.setValue("")` and at that time it throws excepation.
